app/post/shared/post-metadata.service.js

```diff
@@ -0,0 +1,35 @@
+module.exports = [
```

Comment:
Could this file use the newer style too? https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md
Primarily use named function - so you can trace them in the debugger, and bind to the service at the top of the funciton..